### PR TITLE
System contract dapp playground

### PR DESCRIPTION
**Pull request by quiet-node**
**Description**: This PR adds an initialization of the Hedera System Contract Dapp Playground to the project
* Project setup by nextjs@13.4.12
* Configured with
     + TailwindCSS
     + ChakraUI
     + Basic SEO metadata
     + Styrene A Web fonts
     + Hedera branding colors

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: system-contract-dapp-playground/package.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+  "name": "system-contract-dapp-playground",
```
Comment (Nana-EC) [inline system-contract-dapp-playground/package.json:2]: Take a look at the package.json file in the root dir to see the format of fields and other fields to fill in

File: system-contract-dapp-playground/README.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+### Requirements
+
+- [git](https://git-scm.com/)
+- [npm](https://www.npmjs.com/)
+- [nodejs](https://nodejs.org/en/)
+- [contributing guide](https://github.com/hashgraph/.github/blob/main/CONTRIBUTING.md#pull-requests)
+- [Hedera accounts](https://docs.hedera.com/hedera/getting-started/introduction#create-hedera-portal-profile-faucet)
+- [prettier pluggin](https://prettier.io/) (recommended)
+
+## Quickstart
```
Comment (Nana-EC) [inline system-contract-dapp-playground/README.md:23]: Add a `## Config` section and note all the different config files and their purposes.
There's a few more than usual so let's get them noted either here or in the next PR

File: system-contract-dapp-playground/src/fonts/index.ts [hunk 3]
```diff
@@ -0,0 +1,101 @@
+import localFont from 'next/font/local';
```
Comment (Nana-EC) [inline system-contract-dapp-playground/src/fonts/index.ts:1]: All `.ts` files should have a license header.
You can do this in the next PR

File: system-contract-dapp-playground/src/fonts/styreneA-webfont/1802-UFGULA.css [hunk 4]
```diff
@@ -0,0 +1,240 @@
+/*
+        This font software is the property of Commercial Type.
+
+        You may not modify the font software, use it on another website, or install it on a computer.
+
+        License information is available at http://commercialtype.com/eula
+        For more information please visit Commercial Type at http://commercialtype.com or email us at info[at]commercialtype.com
+
+        Copyright (C) 2018 Schwartzco Inc.
+        License: 1802-UFGULA
```
Comment (Nana-EC) [inline system-contract-dapp-playground/src/fonts/styreneA-webfont/1802-UFGULA.css:10]: Erm you might have to remove this. We only want to use open source software. It doesn't sounds like this font package allows
Given the warning I'm not sure how it intends to be used.
What package or tool is this from?

**Review by Nana-EC — CHANGES_REQUESTED** [review]
Nice structure.
Most comments can be addressed in a follow up PR but we should figure out the font situation

File: system-contract-dapp-playground/src/fonts/styreneA-webfont/1802-UFGULA.css (hunk 4 above)
Comment (quiet-node) [reply system-contract-dapp-playground/src/fonts/styreneA-webfont/1802-UFGULA.css:10]: Hello @Nana-EC, thank you for the feedback. I noticed that the Styrene A font is not listed on Google Fonts, and I learned that Hedera employs Styrene A as the primary sans-serif font family. That's why I prefer to stick to this font. These files were downloaded from the Hedera branding site which can be found here https://hederabrandcentral.frontify.com/d/Tmocz52AXpLj/brand-assets#/brand-assets/typography/download-fonts

**Review by Nana-EC — APPROVED** [review]
LG. Can fix up other items in a follow up

**Pull request merged**